crates/vm/levm/src/account.rs

```diff
@@ -48,6 +48,14 @@ impl Account {
         }
     }
 
+    /// New account with info and empty storage
+    pub fn new_with_info(info: AccountInfo) -> Self {
+        Self {
+            info,
+            storage: HashMap::new(),
+        }
+    }
```

Comment:
As an exercise, replace this implementation with `From<AccountInfo> for Account`.

Reply:
Done!

Reply:
https://github.com/lambdaclass/lambda_ethereum_rust/pull/1114/commits/4d0413a2daaa2908c876fc15e769e4f056ff45f9